### PR TITLE
Add Sevio vendor

### PR DESCRIPTION
**Pull request by petrican**
✨ New Vendor

Company site: https://sevio.com/

Sevio is an ad tech company founded in 2014. We focus on building outstanding solutions for web publishers and marketers that help them succeed.

Company address: 
Iasi, Calea Chisinaului
Nr. 104bis-106, Jud. Iasi, Romania

This PR adds and integrates the Sevio Ad Network to the amp vendor list, using the tag amp-ad.

### CODE REVIEW

**Comment by erwinmombay** [comment]
LGTM. will leave final approval from @powerivq

File: ads/_config.js [hunk 1]
```diff
@@ -1189,7 +1189,11 @@ const adConfig = jsonConfiguration({
   'sekindo': {
     renderStartImplemented: true,
   },
-
+  'sevio': {
```
Comment (powerivq) [inline ads/_config.js:1192]: Nit: extra newline before and after.

**Review by powerivq — CHANGES_REQUESTED** [review]
look good otherwise.

File: ads/_config.js (hunk 1 above)
Comment (petrican) [reply ads/_config.js:1193]: Added the newline before and after. Pls check. Thanks

**Comment by petrican** [comment]
@powerivq could you pls review the last pushed changes ? Thanks

**Review by powerivq — APPROVED** [review]

**Pull request merged**